src/functional/fencil_processors/codegens/gtfn/gtfn_ir.py

```diff
@@ -67,24 +67,31 @@ class SymRef(Expr):
 
 
 class Lambda(Expr, SymbolTableTrait):
-    params: List[Sym]
+    params: list[Sym]
     expr: Expr
 
 
 class FunCall(Expr):
     fun: Expr  # VType[Callable]
-    args: List[Expr]
+    args: list[Expr]
 
 
 class FunctionDefinition(Node, SymbolTableTrait):
```

Comment:
Renaming this to `StencilDefinition` probably doesn't make sense because it covers also non-stencil functions, right?

Reply:
Correct, can be anything.